src/compiler/compile/nodes/Element.ts

```diff
@@ -428,7 +428,7 @@ export default class Element extends Node {
 			if (attribute) {
 				const value = attribute.get_static_value();
 
-				if (value === '' || value === '#' || /^\W*?javascript/.test(value)) {
+				if (value === '' || value === '#' || /^\W*javascript:/.test(value)) {
```

Comment:
A case insensitive check should be done, since URI schemes are case insensitive [RFC 2396](https://tools.ietf.org/html/rfc2396#section-3.1).

Reply:
thanks!